usecase/repository/film_repository.go

```diff
@@ -0,0 +1,51 @@
+package repository
+
+import (
+	"bufio"
+	"encoding/csv"
+	"os"
+
+	"github.com/ramrodo/golang-bootcamp-2020/model"
+)
+
+// FilmRepository - interface
+type FilmRepository interface {
+	FindAll() ([]model.Film, error)
+}
+
+// FindAll - reads CSV file and returns an array of Films
+func FindAll() ([]model.Film, error) {
+	films := []model.Film{}
+
+	csvFile, err := os.Open("config/ghibliDB.csv")
```

Comment:
This could be improved later, but you should get the file path from the config file, and also, open the file in the main file and pass the reader object to the layer will use it

Reply:
Ok, I'll read the config in the main file and then, pass it to the other layer